libs/logic-apps-shared/src/designer-client-services/index.ts

```diff
@@ -35,3 +35,4 @@ export * from './lib/userpreference';
 export * from './lib/resource';
 export * from './lib/role';
 export * from './lib/providers/FavoriteContextProvider';
+export * from './lib/vscode';
```

Comment:
Why do we have vscode services in this package? can we keep the service implementation of vscode in vscode package only? this way we wont be adding additional code which will not be used in other portals